feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -92,7 +92,27 @@ class SessionDetailFragment : DaggerFragment() {
             }
 
         binding.bottomAppBar.setOnMenuItemClickListener {
-            handleNavigation(it.itemId)
+            when (it.itemId) {
+                R.id.session_share -> {
+                    // do something
+                }
+                R.id.floormap -> {
+                    handleNavigation(it.itemId)
+                }
+                R.id.session_calendar -> {
+                    val session = binding.session ?: return@setOnMenuItemClickListener false
+                    systemViewModel.openCalendar(
+                        session.title.getByLang(defaultLang()),
+                        session.room.name.getByLang(defaultLang()),
```

Comment:
A room name as a part of event information makes sense. So cool.
Following your idea, I'm thinking attendees would be happy if they could know which room they should enter at a glance.
For example, it could be achieved if a calendar event title is `[<room name>] <session title>`. (in this case, the location column is not required). What do you feel this feedback?

Reply:
Thank you for your feedback.
Your idea sounds fine, but I am wondering that the title of calendar could be too long like below:
> [Hall A] Jetpack時代…

Why don't we just try add the room name at this time, and then touch and try after room name attached?
Fortunately, We can change for anyway.

Reply:
> Why don't we just try add the room name at this time, and then touch and try after room name attached?
> Fortunately, We can change for anyway.

What will we try? and what for change? Could you please rephrase it, sorry?

Reply:
「まずはルーム名を追加して試してみて、ダメだったら変更するようにしませんか？」と言いたかったです

Reply:
Gotcha. I'm fine.